contracts/Challenge/PLCRVotingChallenge/PLCRVotingChallenge.sol

```diff
@@ -0,0 +1,149 @@
+  using SafeMath for uint;
+
+  event ChallengeClosed();
+  event RewardClaimed(uint reward, address voterAddress);
+
+  address public challenger;             // address of the challenger
+  address public listingOwner;           // address of the listingOwner
+  uint public challengerStake;           // amount of tokens challenger staked in challenge
+  TokenCuratedRegistry public registry;  // address of registry
+  PLCRVoting public voting;              // address of PLCRVoting Contract
+  uint public pollID;                    // poll ID for PLCRVoting contract
+  uint public rewardPool;                // pool of tokens to be distributed to winning voters
+  bool public isClosed;                  // signifies whether challenge has been closed
+  uint public voterTokensClaimed;        // total amount of winning tokens voters have received rewards for thus far
+  uint public voterRewardsClaimed;       // total amount of rewards distributed to voters thus far
+
+  mapping(address => bool) public tokenClaims;   // Indicates whether a voter has claimed their reward yet
+
+  // @dev create new challenge
+  // @param _challenger           account that challenged registry item
+  // @param _listingOwner         account of listingOwner
+  // @param _challengerStake      amount challenger staked for challenge
+  // @param _registry             registry contract
+  // @param _plcrVoting           PLCRVoting contract with voting poll
+  // @param _commitStageLength    commitStageLength for PLCRVoting poll
+  // @param _revealStageLength    revealStageLength for PLCRVoting poll
+  // @param _voteQuorum           voteQuorum needed for winning outcome
+  // @param _percentVoterReward   percent of challengerStake rewarded to voters
+  constructor (
+    address _challenger,
+    address _listingOwner,
+    uint _challengerStake,
+    address _registry,
+    address _plcrVoting,
+    uint commitStageLength,
+    uint revealStageLength,
+    uint voteQuorum,
+    uint _percentVoterReward
+  ) public {
+    require(_percentVoterReward <= 100);
+    challenger = _challenger;
+    listingOwner = _listingOwner;
+    challengerStake = _challengerStake;
+    registry = TokenCuratedRegistry(_registry);
+    voting = PLCRVoting(_plcrVoting);
+    pollID = voting.startPoll(voteQuorum, commitStageLength, revealStageLength);
+    rewardPool = (_percentVoterReward.mul(challengerStake)).div(100);
+  }
+
+  // ====================
+  // CHALLENGE INTERFACE:
+  // ====================
+
+  // @notice Close challenge
+  // @dev Closes challenge if PLCRVoting poll has ended and challenge
+  //      is not yet closed
+  function close() public {
+    require(voting.pollEnded(pollID) && !isClosed);
+    isClosed = true;
+
+    require(registry.token().approve(registry, reward()));
+    emit ChallengeClosed();
+  }
+
+  // @notice Determines if the challenge has passed
+  // @dev Check if votesAgainst out of totalVotes exceeds votesQuorum (requires ended)
```

Comment:
not sure this comment applies anymore since this is all handled in PLCR. Maybe just say `returns true if PLCR voting has passed, and false if PLCR voting has not passed. Reverts if challenge has not been closed.`